src/languages/en.js

```diff
@@ -178,6 +178,13 @@ export default {
             invalidSplit: 'Splits amount does not equal total amount',
             other: 'Unexpected error, please try again later',
         },
+        transactions: {
+            cancel: ({amount, comment}) => `Cancelled the ${amount} request${comment ? ` for ${comment}` : ''}.`,
+            create: ({amount, participant, comment}) => `Requested ${amount} from ${participant}${comment ? ` for ${comment}` : ''}.`,
+            decline: ({amount}) => `Declined the ${amount} request.`,
```

Comment:
Can we add the comment here?

```suggestion
            decline: ({amount, comment}) => `Declined the ${amount} request for ${comment}.`,
```